### PR TITLE
Fixes OBC page crash issue

### PR DESCRIPTION
**Pull request by bipuladh**
Updates openshift-sdk to 0.0.21
Updates imports of useActiveNamespace
Makes changes to Modals as required

### CODE REVIEW

File: packages/shared/src/modals/EditAnnotations.tsx [hunk 1]
```diff
@@ -93,7 +91,7 @@ export const AnnotationsModal: React.FC<AnnotationsModalProps> = ({
   );
   return (
     <Modal
-      isOpen={isOpen}
+      isOpen
```
Comment (GowthamShanmugam) [inline packages/shared/src/modals/EditAnnotations.tsx:94]: How the close modal has been controlled with passing isOpen?

File: packages/shared/src/modals/EditLabelModal.tsx [hunk 2]
```diff
@@ -29,16 +29,13 @@ type Patch = {
...
   resource: K8sResourceCommon;
   resourceModel: K8sModel;
   cluster?: string;
 };
 
-type EditLabelModalProps = {
-  labelClassName?: string;
-} & CommonModalProps<EditLabelModalExtraProps>;
-
 export const arrayify = (obj) => _.map(obj, (v, k) => (v ? `${k}=${v}` : k));
```
Comment (GowthamShanmugam) [inline packages/shared/src/modals/EditLabelModal.tsx:39]: So the CommonModalProps is removed becuase it has only extraProps now?

File: package.json [hunk 3]
```diff
@@ -55,7 +55,7 @@
     "dev:c": "yarn ocp-console & PLUGIN=${PLUGIN} I8N_NS=${I8N_NS} yarn server:plugin & wait"
   },
   "dependencies": {
-    "@openshift-console/dynamic-plugin-sdk": "0.0.19",
+    "@openshift-console/dynamic-plugin-sdk": "0.0.21",
```
Comment (GowthamShanmugam) [inline package.json:58]: OCP 4.14 is compatible with this SDK version, right?

Comment (GowthamShanmugam) [reply package.json:58]: no need to update the SDK version on Shared/package.json?

File: packages/shared/src/details-page/DetailsPage.tsx [hunk 4]
```diff
@@ -314,6 +314,8 @@ export const ResourceSummary: React.FC<ResourceSummaryProps> = ({
 
   const launchModal = useModal();
 
+  const launchAnnotationaModal = useAnnotationsModal(resource);
```
Comment (SanjalKatiyar) [inline packages/shared/src/details-page/DetailsPage.tsx:317]:
```suggestion
  const launchAnnotationsModal = useAnnotationsModal(resource);
```

**Comment by SanjalKatiyar** [comment]
> /retitle Fixes OBC page crash issue

removing the BZ number from title because we need to backport this to 4.14 as well, if this PR is merged it can happen that BZ will move to MODIFIED before even we create/merge a backport...

**Review by GowthamShanmugam — COMMENTED** [review]
LGTM

**Comment by SanjalKatiyar** [comment]
/override /test odf-console-e2e-aws
PR is tested locally, overriding the tests as they need OCP 4.14 to pass (currently this PR is running on OCP 4.13)... need to make corresponding change in `openshift/release` repo and for that this PR needs to get merged (otherwise PR on `release` side won't pass)...

**Comment by SanjalKatiyar** [comment]
> /override /test odf-console-e2e-aws PR is tested locally, overriding the tests as they need OCP 4.14 to pass (currently this PR is running on OCP 4.13)... need to make corresponding change in `openshift/release` repo and for that this PR needs to get merged (otherwise PR on `release` side won't pass)...

/override ci/prow/odf-console-e2e-aws

**Pull request merged**